roles/contiv_network/tasks/main.yml

```diff
@@ -41,13 +29,20 @@
 - name: install contivctl
   shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
 
-- name: install docker-compose
-  shell: pip install docker-compose
+- yum: name=python-pip
```

Comment:
I think the task name went missing here. It's just a convention that we have been following to add task names, so may be we should just follow that here too.

Also slightly unrelated, and might not be in the purview of this PR. But I think we can move the docker-compose related installation tasks to the docker role. Would you be ok to do that as part of this PR?